packages/fiori/src/DynamicSideContent.js

```diff
@@ -0,0 +1,546 @@
+ *
+ *
+ * <h3>Responsive Behavior</h3>
+ *
+ * Screen width > 1440px
+ *
+ * <ul><li>Main vs. side content ratio is 75 vs. 25 percent (with a minimum of 320px
+ * each).</li>
+ * <li>If the application defines a trigger, the side content can be hidden.</li></ul>
+ *
+ * Screen width <= 1440px and > 1024px
+ *
+ * <ul><li>Main vs. side content ratio is 66.666 vs. 33.333 percent (with a minimum of
+ * 320px each). If the side content width falls below 320 px, it automatically slides
+ * under the main content, unless the app development team specifies that it should
+ * disappear.</li></ul>
+ *
+ * Screen width <= 1024px and > 720px
+ *
+ * <ul><li>If <code>sideContentFallDown</code> is not set to <code>OnMinimumWidth</code>,
+ * side content ratio is fixed to 340px, and the main content takes the rest of the width,
+ * otherwise the above ratios are valid only for screen width <= 1024px and > 960px, and
+ * for screen width <= 960px and > 720px the side content falls below the main content.
+ * </li></ul>
+ *
+ * Screen width <= 720px (for example on a mobile device)
+ *
+ * <ul><li>In this case, the side content automatically disappears from the screen (unless
+ * specified to stay under the content by setting of <code>sideContentVisibility</code>
+ * property to <code>AlwaysShow</code>) and can be triggered from a pre-set trigger
+ * (specified within the app). When the side content is triggered, it replaces the main
+ * content. We recommend that you always place the trigger for the side content in the
+ * same location, such as in the app footer.</li></ul>
+ *
+ * A special case, allows for comparison mode between the main and side content. In
+ * this case, the screen is split into 50:50 percent for main vs. side content. The
+ * responsive behavior of the equal split is the same as in the standard view - the
+ * side content disappears on screen widths of less than 720 px and can only be
+ * viewed by triggering it.
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/DynamicSideContent";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.DynamicSideContent
+ * @extends UI5Element
+ * @tagname ui5-dynamic-side-content
+ * @public
+ */
+class DynamicSideContent extends UI5Element {
+	constructor() {
+		super();
+		this._handleResizeBound = this.handleResize.bind(this);
+	}
+
+	static get metadata() {
+		return metadata;
+	}
+
+	static get styles() {
+		return DynamicSideContentCss;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get template() {
+		return DynamicSideContentTemplate;
+	}
+
+	static async onDefine() {
+		DynamicSideContent.i18nBundle = await getI18nBundle("@ui5/webcomponents-fiori");
+	}
+
+	get classes() {
+		const gridPrefix = "ui5-dsc-span",
+			mcSpan = this._toggled ? this._scSpan : this._mcSpan,
+			scSpan = this._toggled ? this._mcSpan : this._scSpan,
+			classes = {
```

Comment:
keys should ideally be just `root`, `main` and `side` as we don't repeat the class name in class member variables as a rule (same analogy applies here)

Reply:
Done